jdk/make/closed/autoconf/custom-spec.gmk.in

```diff
@@ -56,6 +56,10 @@ USERNAME                := @USERNAME@
 OPENJDK_SHA             := @OPENJDK_SHA@
 OPENJDK_TAG             := @OPENJDK_TAG@
 
+JDK_MOD_VERSION:=@JDK_MOD_VERSION@
+JDK_FIX_VERSION:=@JDK_FIX_VERSION@
+J9JDK_EXT_VERSION:=${JDK_MINOR_VERSION}.${JDK_MICRO_VERSION}.${JDK_MOD_VERSION}.${JDK_FIX_VERSION}
```

Comment:
Can we have some (at least one) space either side of `:=`, please, for consistency?